Jenkinsfile

```diff
@@ -1,29 +1,30 @@
 elifePipeline {
     def commit
+    stage 'Checkout', {
+        checkout scm
+        commit = elifeGitRevision()
+    }
+
     elifeOnNode(
         {
-            stage 'Checkout', {
+            stage 'Container image', {
                 checkout scm
```

Comment:
Jenkins performs everything inside `elifeOnNode()` on a separate node, that's why we need an additional `checkout scm` that performs the git clone on this node too. In the future we could do everything on this `elife-libraries--...` node, but for now the rest of the build after `ci` still needs to be executed on the Jenkins main node.